From 47365da6c9254974bf314b043b9b530b823a657e Mon Sep 17 00:00:00 2001
From: Guanhua Gao <guanhua.gao@nxp.com>
Date: Wed, 30 May 2018 18:20:43 +0800
Subject: [PATCH 534/767] staging: fsl-dpaa2/eth: Use TC_SETUP_MQPRIO to
 compatible with the kernel version

Use TC_SETUP_MQPRIO to compatible with the kernel version.

Signed-off-by: Guanhua Gao <guanhua.gao@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
index e460d6bdf62c..99823bbc42a8 100644
--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
@@ -1953,7 +1953,7 @@ static int dpaa2_eth_setup_tc(struct net_device *net_dev,
 	struct tc_mqprio_qopt *mqprio = (struct tc_mqprio_qopt *)type_data;
 	int i, err = 0;
 
-	if (type != TC_SETUP_QDISC_MQPRIO)
+	if (type != TC_SETUP_MQPRIO)
 		return -EINVAL;
 
 	if (mqprio->num_tc > dpaa2_eth_tc_count(priv)) {
-- 
2.17.0

